app/questionnaire/rules.py

```diff
@@ -62,6 +62,15 @@ def evaluate_condition(condition, answer_value, match_value):
     """
     answer_and_match = answer_value is not None and match_value is not None
 
+    if condition in {"equals", "not equals", "equals any", "not equals any"}:
```

Comment:
Do we need to cater for all the 'contains'? I guess we have more control over them being pre-determined lists.

Reply:
Yeah was decided not to look at the contains, as we define the lists